demo/hashlist.redis

```diff
@@ -0,0 +1,9 @@
+HSET OP_KEYS_WHITELIST "707B6451B8319C28E412F847E17BB87995441AF356007A03A3A4AC7745A5223D" 1
```

Comment:
Is there a way to put a comment in this file explaining which hash pertains to which certificate?

Reply:
Done